### PR TITLE
[fix] Fix storageSize*/ offloadedStorageSize* stats descriptions.

### PR DESCRIPTION
**Pull request by dao-jun**
### ✅ Contribution Checklist

- [x] I read the [contribution guide](https://pulsar.apache.org/contribute/document-contribution/)
- [x] I updated the [versioned docs](https://pulsar.apache.org/contribute/document-contribution/#update-versioned-docs)

### CODE REVIEW

File: docs/administration-stats.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ All stats below are **reset** to 0 upon broker restart or topic unloading, **exc
...
 | averageMsgSize                                   | The average size (bytes) of messages published within the last interval.                                                                                                                                                                        |
-| storageSize*                                     | The sum of the ledgers' storage size **in BookKeeper** for a topic (in bytes). <br/><br/>**Note**: the `total storage size of a topic` = `storageSize` + `offloadedStorageSize`.                                                                |
-| offloadedStorageSize*                            | The sum of the storage size **in tiered storage** for a topic (in bytes).<br/><br/>**Note**: the `total storage size of a topic` = `storageSize` + `offloadedStorageSize`.                                                                      |
+| storageSize*                                     | The sum of the ledgers' storage size **in BookKeeper** and **in tiered storage** for a topic (in bytes). <br/><br/>**Note**: the `total storage size of a topic` = `storageSize`.                                                               |
```
Comment (asafm) [inline docs/administration-stats.md:25]: Maybe we don't need the `**Note**` anymore.

Comment (dao-jun) [reply docs/administration-stats.md:25]: Yes, for storageSize, we don't need it anymore

**Review by asafm — APPROVED** [review]

**Comment by asafm** [comment]
@dao-jun Before merging, can you please double check each section looks ok (renders correctly) via the preview?

**Comment by dao-jun** [comment]
image

**Comment by dao-jun** [comment]
merging...

**Pull request merged**